charts/prefect-server/values.schema.json

```diff
@@ -774,7 +774,7 @@
           "description": "enable ingress record generation for server"
         },
         "servicePort": {
-          "type": "string",
+          "type": "integer",
```

Comment:
I took a look at this, and we use a [Bitnami `common` chart helper](https://github.com/bitnami/charts/blob/d1ea2593fc3a34439b742ac077edec74a1b8b3a6/bitnami/common/templates/_ingress.tpl#L8-L34) to define the backend. There, you can see it has conditional logic to consider if the `servicePort` value is a string or an integer (a port name or a port number).

So we can actually allow both here (and update the description):

```json
        "servicePort": {
          "type": [
            "integer",
            "string",
          ],
          "title": "Service Port",
          "description": "service port name or number"
        },
```